examples/with-create-react-app/apps/web/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "web",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "dev": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "TIMING=1 eslint src --fix",
+    "clean": "rm -rf build"
+  },
+  "dependencies": {
+    "ui": "*",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1",
```

Comment:
Do we need react-router?

Reply:
Yea interestingly this ships with CRA, but it's not used in the example page so we can definitely yank it.

Reply:
I'm generally opposed to changing much that comes out-of-the-box with contents of a generator—even if unused. We want the upstream documentation about how to do things to still apply. Dropping this just trolls somebody who is getting started with CRA and doesn't know why their thing doesn't work.

Reply:
That's a fair point. Generally speaking I did my best to leave things that started from a generator alone for this exact reason - we don't want to have to support "turbo's CRA". So I would be fine putting this back, but it's also a pretty easy error to see and fix if someone does run into this down the line.